compiler/mx.compiler/suite.py

```diff
@@ -4,8 +4,8 @@
   "sourceinprojectwhitelist" : [],
 
   "groupId" : "org.graalvm.compiler",
-  "version" : "22.3.0.1",
-  "release" : True,
+  "version" : "22.3.0.2",
```

Comment:
Shouldn't this be `22.3.1` for January 2023?

Reply:
Yes, but we might need to release `22.3.0.2` before that.

Reply:
Fair enough.